.github/workflows/_runner-determinator.yml

```diff
@@ -717,18 +717,4 @@ jobs:
       - name: Get the workflow type for the current user
         id: set-condition
         run: |
-          curr_branch="${{ inputs.curr_branch }}"
```

Comment:
I guess this is just a temp change to bypass the recent issue with `no-runner-experiment`?